components/sts/org.wso2.carbon.identity.sts.mgt.ui/src/main/resources/web/identity-sts/configure.jsp

```diff
@@ -118,11 +119,11 @@ try {
 				<table class="normal" cellspacing="0">
 		          <tr>
 	                      <td><fmt:message key='card.name'/><font class="required">*</font> </td>
-	                      <td> <input type="text" name="cardName" id="cardName" value="<%=cardIssuer.getCardName()%>" /></td>	                     
+	                      <td> <input type="text" name="cardName" id="cardName" value="<%=Encode.forHtmlAttribute(cardIssuer.getCardName())%>" /></td>
 	                  </tr>
 	                  <tr>
 	                      <td><fmt:message key='valid.period'/><font class="required">*</font> </td>  
-	                      <td><input type="text" name="validPeriod" id="validPeriod" value="<%=cardIssuer.getValidPeriodInDays()%>" /></td>                  
+	                      <td><input type="text" name="validPeriod" id="validPeriod" value="<%=cardIssuer.getValidPeriodInDays()%>" /></td>
```

Comment:
cardIssuer.getValidPeriodInDays() returns an integer, hence doesn't need to Encode.